include/verilated_timing.h

```diff
@@ -0,0 +1,367 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//*************************************************************************
+//
+// Code available from: https://verilator.org
+//
+// Copyright 2003-2022 by Wilson Snyder. This program is free software; you can
+// redistribute it and/or modify it under the terms of either the GNU
+// Lesser General Public License Version 3 or the Perl Artistic License
+// Version 2.0.
+// SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+//
+//*************************************************************************
+#ifndef VERILATOR_VERILATED_TIMING_H_
+#define VERILATOR_VERILATED_TIMING_H_
+
+// Dynamic scheduling-related includes
+#include <functional>
+#include <queue>
+#include <unordered_map>
+#include <unordered_set>
+#include <utility>
+#include <vector>
+
+// Some preprocessor magic to support both Clang and GCC coroutines with both libc++ and libstdc++
+#ifdef __clang__
+#ifdef _LIBCPP_VERSION
+// Using libc++, coroutine library is in std::experimental
+#include <experimental/coroutine>
+namespace std {
+// Bring std::experimental into the std namespace
+using namespace experimental;
```

Comment:
This seems a little risky, maybe we should only use those namespace elements that are needed in this file?